tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -206,16 +206,17 @@ def langchain_local_model_dir(tmp_path):
         "artifact_path": "some-artifact-path",
         "run_id": "abc123",
         "signature": fake_signature.to_dict(),
-        "flavors": {
-            "langchain": {
-                "databricks_dependency": {
-                    "databricks_vector_search_index_name": ["index1", "index2"],
-                    "databricks_embeddings_endpoint_name": ["embedding_endpoint"],
-                    "databricks_llm_endpoint_name": ["llm_endpoint"],
-                    "databricks_chat_endpoint_name": ["chat_endpoint"],
-                }
+        "resources": {
```

Comment:
Lets ensure Serving side changes are supported and released before mlflow release is finished